updater/UPDATER_DEMO.md

```diff
@@ -0,0 +1,128 @@
+To replicate the updater demo, you'll need a copy of the Flutter Engine.
```

Comment:
Most of this should hopefully be able to go away very soon :)

Reply:
Correct.